APIManager.js

```diff
@@ -0,0 +1,80 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {}
+    }
+    loadUser() {
+       const result =  $.ajax({
```

Comment:
you do not need to save the ajax call inside a variable. You are already saving the data inside this.data

if you used the success option in ajax you would not need to do this

you only create a variable if you use it later in your code.